### ISSUE TITLE
Failing on Terraform 0.15

### ISSUE THREAD

**Issue by abbaspour** [squidfunk/terraform-aws-api-gateway-enable-cors#12]
Terraform 0.15 removed the deprecated `map()` function, hence this module is not working. Error log below:

```
╷
│ Error: Error in function call
│
│   on .terraform/modules/cors_policy_item/headers.tf line 27, in locals:
│   27:   headers = map(
│   28:     "Access-Control-Allow-Headers", "'${join(",", var.allow_headers)}'",
│   29:     "Access-Control-Allow-Methods", "'${join(",", var.allow_methods)}'",
│   30:     "Access-Control-Allow-Origin", "'${var.allow_origin}'",
│   31:     "Access-Control-Max-Age", "'${var.allow_max_age}'",
│   32:     "Access-Control-Allow-Credentials", var.allow_credentials ? "'true'" : ""
│   33:   )
│     ├────────────────
│     │ var.allow_credentials will be known only after apply
│     │ var.allow_headers is a list of string, known only after apply
│     │ var.allow_max_age is a string, known only after apply
│     │ var.allow_methods is a list of string, known only after apply
│     │ var.allow_origin is a string, known only after apply
│
│ Call to function "map" failed: the "map" function was deprecated in Terraform v0.12 and is no longer available; use tomap({ ... }) syntax to write a literal map.
```

**Comment by squidfunk** [comment]
Fixed by #13.

**Comment by abbaspour** [comment]
build is failing. 0.3.3 not published :(

**Comment by squidfunk** [comment]
I see, some things seem to have changed. Might you be able to help fix the build in a PR? I can re-issue 3.3 then.

**Comment by squidfunk** [comment]
I just reissued 0.3.3 after the fixes in #14.
